src/app/components/media/MediaLanguageSwitcher.js

```diff
@@ -70,7 +70,7 @@ const MediaLanguageSwitcher = ({ projectMedia, setFlashMessage }) => {
       label={<FormattedMessage id="mediaLanguageSwitcher.selectLanguageLabel" defaultMessage="Language" description="Label for input to select language" />}
       selectedLanguage={projectMedia.language_code}
       onSubmit={handleUpdate}
-      languages={safelyParseJSON(projectMedia.team?.get_languages)}
+      languages={projectMedia.team ? safelyParseJSON(projectMedia.team.get_languages) : [projectMedia.language_code]}
```

Comment:
What happens if there's neither `projectMedia.team` nor `projectMedia.language_code`? Do we need to check for that case as well?

Reply:
Hum... sure... I added another fallback... if there's neither `projectMedia.team` nor `projectMedia.language_code`, I'm now falling back to `und`.